test/functional/p2p_sendtxrcncl.py

```diff
@@ -39,6 +39,12 @@ def __init__(self):
     def on_sendtxrcncl(self, message):
         self.sendtxrcncl_msg_received = message
 
+
+class P2PFeelerReceiver(SendTxrcnclReceiver):
+    def on_version(self, message):
+        pass  # feeler connections can not send any message other than their own version
```

Comment:
nit, if you have to retouch
```suggestion
        pass  # feeler connections cannot send any message other than their own version
```